portfolio/src/main/webapp/script.js

```diff
@@ -15,29 +15,95 @@
 /**
  * Adds a random greeting to the page.
  */
-function addRandomGreeting() {
-  const greetings =
-      ['Simpsons', 'Gossip Girl', 'Community', 'Emily in Paris'];
+showHasBeenClicked = false;
+factHasBeenClicked = false;
+function recommendRandomShow() {
+  const shows =
+    ['The Good Place', 'Gossip Girl', 'Community', 'Emily in Paris', 'How I met Your Mother'];
+    
+  // Pick a random tv show.
+  const show = shows[Math.floor(Math.random() * shows.length)];
 
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
+  // Add it to the page.
+  const showContainer = document.getElementById('tv-show-container');
+  
+  if (showHasBeenClicked === true){
+    showContainer.innerText = null;
+    showHasBeenClicked = false;
+  }
+    
+  else{
+      showContainer.innerText = show;
+      showHasBeenClicked = true;
+  }
+    
+}
+/**
+ * Adds a random fact to the page.
+ */
+function randomFact() {
+  const facts =
+    ['favourite color - Purple', 'prefers dogs over cats', 'favourite ice cream flavor - strawberry', 
+    'favourite singer - Lady Gaga', 'still dreams to become an astronauts', '90% of times remembers her dreams'];
+
+  // Pick a random fact.
+  const fact = facts[Math.floor(Math.random() * facts.length)];
 
   // Add it to the page.
-  const greetingContainer = document.getElementById('greeting-container');
-  greetingContainer.innerText = greeting;
+  const factContainer = document.getElementById('fact-container');
+    if (factHasBeenClicked === true){
+      factContainer.innerText = null;
+      factHasBeenClicked = false;
+    }
+  else{
+      factContainer.innerText = fact;
+      factHasBeenClicked = true;
+  }
+}
+//Button
```

Comment:
Can you give this a more descriptive function comment please.

Reply:
added to the new commit